src/Assets/TestProjects/BlazorHosted/blazorhosted/blazorhosted.csproj

```diff
@@ -8,6 +8,11 @@
     <ProjectReference Include="..\blazorwasm\blazorwasm.csproj" />
   </ItemGroup>
 
+  <!-- Based on .NET 8 breaking change: Add SelfContained if RuntimeIdentifier is defined to match old behavior.. -->
+  <PropertyGroup Condition="'$(RuntimeIdentifier)' != '' and '$(SelfContained)' == ''">
+    <SelfContained>true</SelfContained>
+  </PropertyGroup>
```

Comment:
Unfortunately, it looks like there are still edge cases where Blazor would be broken so I am coordinating with them. I don't think we want to release this and break Blazor in the process so it will probably get delayed to preview 5 unless we can do it quickly.

cc @javiercn @danroth27